src/napari_matplotlib/base.py

```diff
@@ -125,25 +132,31 @@ def draw(self) -> None:
         This is a no-op, and is intended for derived classes to override.
         """
 
-    def apply_napari_colorscheme(self) -> None:
-        """Apply napari-compatible colorscheme to the axes object."""
-        if self.axes is None:
-            return
+    def add_single_axes(self) -> None:
+        """
+        Add a single Axes to the figure.
+
+        The Axes is saved on the ``.axes`` attribute for later access.
+        """
+        self.axes = self.figure.subplots()
+        self.apply_napari_colorscheme(self.axes)
+
+    @staticmethod
+    def apply_napari_colorscheme(ax: Axes) -> None:
+        """Apply napari-compatible colorscheme to an axes object."""
         # changing color of axes background to transparent
-        self.canvas.figure.patch.set_facecolor("none")
```

Comment:
~The figure's facecolor doesn't need to be set?~

No, it doesn't but I don't get why.

Reply:
It's already set in `NapariMPLWidget.__init__()`